core/models.py

```diff
@@ -623,7 +627,17 @@ def clean_associated_data_base_table(sender, instance, **kwargs):
 post_delete.connect(clean_associated_data_base_table, sender=DataTable)
 
 
+class TableFileQuerySet(models.QuerySet):
+    def most_recent_for_table(self, table):
+        table_file = self.filter(table=table).first()
+        if not table_file:
+            raise TableFile.DoesNotExist(f"For table {table}")
+        return table_file
```

Comment:
Usar `get_` ficaria mais padronizado:

```suggestion
    def for_table(self, table):
        return self.filter(table=table)
        
    def get_most_recent(self):
        qs = self.order_by("-created_at")
        return qs.get()
```

Reply:
Coloquei o `get_` no commit ac71e03 mas preferi não quebrar em dois métodos porque:

1. Quem for usar o queryset vai ter que saber dos dois métodos e ter mais trabalho;
2. A mensagem de erro no caso de não achar o mais recente fica ruim de entender porque não tem o nome da tabela que foi utilizada;